db_stress_tool/db_stress_test_base.cc

```diff
@@ -309,7 +310,11 @@ void StressTest::FinishInitDb(SharedState* shared) {
       exit(1);
     }
   }
-
+#ifndef ROCKSDB_LITE
+  if (FLAGS_use_txn) {
+    ProcessRecoveredPreparedTxns(shared);
+  }
```

Comment:
Need comment that this is OK here without sync because unsynced data cannot be lost at this point - even with sync_fault_injection=1, the file is still directly writable until after `FinishInitDb()`

Reply:
Sounds good!

Reply:
Fixed